components/x11/xorg-server/patches/35-gcc-10.patch

```diff
@@ -0,0 +1,78 @@
+--- xorg-server-1.19.7/randr/rrprovider.c.orig	2020-05-14 10:17:01.138670184 +0000
++++ xorg-server-1.19.7/randr/rrprovider.c	2020-05-14 10:17:12.964330073 +0000
+@@ -27,7 +27,7 @@
+ 
+ #include <X11/Xatom.h>
+ 
+-RESTYPE RRProviderType;
++RESTYPE RRProviderType = 0;
+ 
+ /*
+  * Initialize provider type error value
+--- xorg-server-1.19.7/hw/xwayland/xwayland.c.orig	2020-05-14 09:58:09.466678754 +0000
++++ xorg-server-1.19.7/hw/xwayland/xwayland.c	2020-05-14 09:58:17.105153183 +0000
+@@ -37,7 +37,7 @@
+ 
+ #ifdef XF86VIDMODE
+ #include <X11/extensions/xf86vmproto.h>
+-_X_EXPORT Bool noXFree86VidModeExtension;
++extern _X_EXPORT Bool noXFree86VidModeExtension;
+ #endif
+ 
+ void
+--- xorg-server-1.19.7/hw/dmx/dmxextension.c.orig	2020-05-14 10:13:31.561339683 +0000
++++ xorg-server-1.19.7/hw/dmx/dmxextension.c	2020-05-14 10:24:15.942597405 +0000
+@@ -68,9 +68,6 @@
+  * _any_ header files. */
+ extern FontPtr defaultFont;
+ 
+-/* Hack to get Present to build (present requires RandR) */
+-RESTYPE RRCrtcType;
+-
```

Comment:
Is RRCrtcType not needed anymore?

Reply:
It is actually not used in this source and redeclared + used in librandr.a